sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_shared/async_client_base.py

```diff
@@ -4,11 +4,12 @@
 # ------------------------------------
 from typing import TYPE_CHECKING
 
-from azure.core.pipeline import AsyncPipeline
+from azure.core.pipeline.transport import AioHttpTransport
```

Comment:
We should import this only when we need to use it, because `aiohttp` isn't required to install or use the library.